addons/l10n_tr_nilvera_einvoice/models/account_move.py

```diff
@@ -316,6 +316,23 @@ def _l10n_tr_nilvera_add_pdf_to_invoice(self, client, invoice, document_uuid, do
         invoice.message_main_attachment_id = attachment
         invoice.with_context(no_new_invoice=True).message_post(attachment_ids=attachment.ids)
 
+    def l10n_tr_nilvera_get_pdf(self):
+        with _get_nilvera_client(self.env.company) as client:
+            for invoice in self:
+                if (
+                        invoice.l10n_tr_nilvera_customer_status not in {'einvoice','earchive'}
```

Comment:
```suggestion
                        invoice.l10n_tr_nilvera_customer_status not in {'einvoice', 'earchive'}
```
This is breaking the styling 😄